python/setup.py

```diff
@@ -20,91 +17,17 @@
 from setuptools.command.develop import develop
 from setuptools.command.test import test as test_command
 
-PLATFORM = "unix"
-if platform.platform().startswith("Win"):
-    PLATFORM = "win"
 
-MODEL_DIR = os.path.join("stan", PLATFORM)
```

Comment:
Yeah I'm not sure what the history is behind having different model code for Windows, but if the compilation and tests pass on Windows then happy to make this simplifying change.